new-project/src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from "./logo.svg";
-import "./App.css";
+import Example from "./components/Example";
+import AnotherExample from "./components/AnotherExample";
+import ArrayChanger from "./components/Arrays/Array";
+import FormWithHook from "./components/UserInputFormExample";
+import FormWithRef from "./components/UserInputFormExampleRef";
+import User from "./components/SandboxComponent";
```

Comment:
Check your components naming. Components should be called by their main functionality